pkg/minikube/machine/client.go

```diff
@@ -152,8 +154,25 @@ func CommandRunner(h *host.Host) (command.Runner, error) {
 	if driver.BareMetal(h.Driver.DriverName()) {
 		return command.NewExecRunner(), nil
 	}
+	sshRunner := command.NewSSHRunner(h.Driver)
```

Comment:
This needs to be moved to a separate PR. I'm not entirely sure why this is here at the moment, and would like to  hear more about it's impact to start latency.

Reply:
I dont return the kic runner, I only use it once for verifying if the SSH service is UP